VTCareService/src/main/java/edu/vt/cs/vtcare/vtcareservice/dao/ProviderDao.java

```diff
@@ -13,7 +13,7 @@ public ProviderDao() throws Exception {
     }
 
     private static final String CREATE_PROVIDER_SQL =
-            "INSERT INTO providers (provider_name, email, provider_password, gender, date_of_birth, address, contact, specialization, experience)" +
+            "INSERT INTO providers (name, email, password, gender, date_of_birth, address, contact, specialization, experience)" +
```

Comment:
Only 2 columns with provider_ prefix? I hope you said 'most' columns are with provider_ Please be accurate in your observations. 2/8 columns is simple elementary level maths and doesn't qualify for 'most' :)))

Reply:
Please add fruitful comments here.

Reply:
I was just clarifying your mistake in observation. We are engineers, we need to be 'specific'.